File: 2015/src/test/java/nl/sbmf21/aoc15/days/Day9Test.java
package nl.sbmf21.aoc15.days;

import org.junit.jupiter.api.Test;

import static nl.sbmf21.aoc.testing.UtilKt.testDay;

public class Day9Test {

    @Test
    public void testInput() {
        testDay(Day9.class, 251, 898, false, null);
    }

    @Test
    public void testExample() {
        testDay(Day9.class, 605, 982, true, null);
    }
}

File: 2023/kotlin/src/test/kotlin/nl/sbmf21/aoc23/days/Day9Test.kt
package nl.sbmf21.aoc23.days

import nl.sbmf21.aoc.testing.testDay
import org.junit.jupiter.api.Test

class Day9Test {

    @Test
    fun testInput() = testDay(Day9::class.java, 1_581_679_977L, 889L)

    @Test
    fun testExample() = testDay(Day9::class.java, 114L, 2L, true)
}